torch/_decomp/decompositions.py

```diff
@@ -2761,6 +2762,33 @@ def upsample_bicubic2d_vec(
     return upsample_bicubic2d_default(a, output_size, align_corners, scale_h, scale_w)
 
 
+@register_decomposition(torch.ops.quantized_decomposed.quantize_per_tensor)
+def quantize_per_tensor(
```

Comment:
n00b question, what's the difference between the original function and this function?  They both seem to do the same thing, with the math in Python.  Are there some systems that treat functions in `torch/_decomp/decompositions.py` differently?  Note: I've never used this file.

Reply:
I think they serve different purposes, the original function is for execution, the function in decomposition is for generating the decomposition for a specific op, although maybe we can just use a single implementation